packages/enterprise/src/license.ts

```diff
@@ -727,6 +727,7 @@ export async function licenseInit(
           if (
             forceRefresh ||
             !mongoCache ||
+            !mongoCache.dateUpdated ||
```

Comment:
In what scenario would this ever happen?

Reply:
In exactly the situation where we never got a reply from the server but errored the very first time.  We save that error in the mongoCache. `dateUpdated` is really `dateUpdatedFromServer`. I'll add a comment.